ethpm/package.py

```diff
@@ -215,5 +222,50 @@ def deployments(self) -> Union["Deployments", Dict[None, None]]:
             )
             for deployment_data in deployments.values()
         }
+        validate_deployments_tx_receipt(deployments, self.w3)
+        linked_deployments = get_linked_deployments(deployments)
+        if linked_deployments:
+            for deployment_data in linked_deployments.values():
+                on_chain_bytecode = self.w3.eth.getCode(
+                    to_checksum_address(deployment_data["address"])
+                )
+                unresolved_linked_refs = normalize_linked_references(
+                    deployment_data["runtime_bytecode"]["link_dependencies"]
+                )
+                resolved_linked_refs = tuple(
+                    self._resolve_linked_references(link_ref, deployments)
+                    for link_ref in unresolved_linked_refs
+                )
+                validate_linked_references(resolved_linked_refs[0], on_chain_bytecode)
```

Comment:
I'm confused about this `[0]` index.  Shouldn't validation happen for everything in that iterable?